tests/restarting/from_7.1.0/VersionVectorEnableRestart-2.toml

```diff
@@ -0,0 +1,54 @@
+[[knobs]]
+enable_version_vector = true
+enable_version_vector_tlog_unicast = true
+
```

Comment:
"PROXY_USE_RESOLVER_PRIVATE_MUTATIONS" not enabled here.

Reply:
It is enabled here.